### PR TITLE
Implements movie and slide detail layout

### LINKED ISSUE #134: Show movie and slide in session detail
## Overview (Required)
- If app can get it link, we should show movie and slide link
image
image

## Links
- https://gallery.io/projects/MCHbtQVoQ2HCZd9Pbpex4PVP/files/MCHtA7U1iMGr60CQdbvgFMCPHe-6adOX1IM (Please send request)
- https://app.swaggerhub.com/apis-docs/jmatsu/DroidKaigi2019/1.0.0#/developers/timtable

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #134  

~~Do not let it close because it is on the way.~~   
~~Close after completing all implementations.~~  

## Overview (Required)
- Implemented movie and slide views.

## ~~Not Implemented(or do not know)~~  
- ~~Action of clicked movie and slide items.(What kind of approach should I implement?)~~  
- ~~English translation.~~  
- ~~Title when there is neither movie nor slide.(hide title?)~~  

## Screenshot
Before | After
:--: | :--:
 | 
 | 
 | 
 | 
　 |

### CODE REVIEW

**Comment by takahirom** [comment]
You can use ActivityActionCreator 🌪 
https://github.com/DroidKaigi/conference-app-2019/pull/253/files#diff-b41d9f19fe508adeeb8de4cd2d9ad6aeR17

**Comment by jmatsu** [comment]
@NUmeroAndDev 
Thank you for your quick action!

> Action of clicked movie and slide items.(What kind of approach should I implement?)

It's okay to just open the url via web

> English translation.

資料 -> Presentaion Materials (if not translated yet)
動画 -> Video
スライド -> Slides

> Title when there is neither movie nor slide.(hide title?)

It would be nice if you just hide the area of the presentation material including the title in this case.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt [hunk 1]
```diff
@@ -122,10 +135,10 @@ abstract class SessionDetailFragmentModule {
             return sessionsFragment.viewLifecycleOwner.lifecycle
         }
 
-        @JvmStatic @Provides fun provideActivity(
-            sessionsFragment: SessionDetailFragment
-        ): FragmentActivity {
-            return sessionsFragment.requireActivity()
-        }
+//        @JvmStatic @Provides fun provideActivity(
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt]: If you not need this function, Please delete it 🙏

**Comment by NUmeroAndDev** [comment]
@takahirom 
@jmatsu 
Thank you for teaching me!
Please check 🙇

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt (hunk 1 above)
Comment (NUmeroAndDev) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt]: @takahirom 
Sorry, I forgot…
Deleted it.

File: model/src/commonMain/kotlin/Session.kt [hunk 2]
```diff
@@ -22,11 +22,16 @@ sealed class Session(
...
         val intendedAudience: String?,
+        val videoUrl: String?,
+        val slideUrl: String?,
         val isInterpretationTarget: Boolean,
         val isFavorited: Boolean,
         val speakers: List<Speaker>,
         val message: SessionMessage?
-    ) : Session(id, dayNumber, startTime, endTime, room)
+    ) : Session(id, dayNumber, startTime, endTime, room) {
+        val hasVideo: Boolean = videoUrl.isNullOrEmpty().not()
```
Comment (takahirom) [inline model/src/commonMain/kotlin/Session.kt]: 🆒

**Comment by takahirom** [comment]
LGTM 🎥

**Pull request merged**